torch/_inductor/output_code.py

```diff
@@ -497,6 +493,7 @@ def post_compile(
                     self,
                     cudagraphs,
                     constants.unwrap(self),
+                    graph_kwargs.get("boxed_forward_device_index", None),
```

Comment:
could we add assertions here - do we know if this is fwd/backward ? if we know its backward, whenever cudagraphs is set this should also be set..

Reply:
Yup will do